book/tables/pharmacokinetic/pkpt02.qmd

```diff
@@ -19,6 +19,24 @@ library(tern)
 
 adpp <- synthetic_cdisc_dataset("latest", "adpp")
 adpp <- adpp %>% filter(PPSPEC == "Plasma", AVISIT == "CYCLE 1 DAY 1")
+
+fmt_sigfig_mean_sd <- function(sigfig) {
```

Comment:
Hmm this is OK but I think it should at least live in `tern` ? Even better if you can make format_sigfig more general so that it can handle multiple input types.

Reply:
@anajens I have added this functionality into the `format_sigfig` function (just for these two formats that we use) within the PR for the `format_sigfig` fix [here](https://github.com/insightsengineering/tern/compare/1066_sigfig_bug%40main?expand=1) if you want to re-review [that PR](https://github.com/insightsengineering/tern/pull/1067).

Reply:
Ok I saw this now. I think we could do some parsing instead of having it fixed with value value_paren etc.. so we are more general.